### PR TITLE
Added images for notes on git

### PR DESCRIPTION
**Pull request by gitskim**

### CODE REVIEW

File: C-Git/git.md [hunk 1]
```diff
@@ -107,6 +111,16 @@ Going back in time:
 
     git checkout <commit hash>
 
+## Examples ##
+
+Case 1: Adding a new file
+
+![git-existing-file.png](../images/git-existing-file.png)
```
Comment (j-hui) [inline C-Git/git.md]: this asks "what happens if I do git diff?" it might be good to answer that question here in these notes. these are supposed to be standalone notes

File: C-Git/git.md [hunk 2]
```diff
@@ -16,6 +16,10 @@ In this class, we require that you make git commits to get full credit for your
 work. We want you to learn git because in industry, you will without a doubt be
 using version control.
 
+## Version Control ##
+
+It is a system that records changes to keep track of versions of a set of files. Programmers no longer have to keep many copies of files on their hard drives manually
```
Comment (j-hui) [inline C-Git/git.md]: can you reword this to

>A Version Control System (VCS) records changes made to a set of files. This allows programmers to maintain backups of their previous work without manually copying their files

File: C-Git/git.md (hunk 1 above)
Comment (gitskim) [reply C-Git/git.md]: I actually decided to remove the question. I will instead ask that question during recitation.
